network/dag/interface.go

```diff
@@ -93,6 +93,11 @@ type State interface {
 	//	- highest lamport clock in the DAG
 	// A requested clock of math.MaxUint32 will return the iblt of the entire DAG
 	IBLT(reqClock uint32) (tree.Iblt, uint32)
+
+	// IncorrectStateDetected is called when the xor and LC value from a gossip message do NOT match the local state.
+	IncorrectStateDetected()
```

Comment:
2 node mismatches = repair?

Reply:
Yes, could become interesting if 2 nodes are out of sync.... Could set it to 5 or even 10